src/bionics.cpp

```diff
@@ -492,7 +492,7 @@ bool player::activate_bionic( int b, bool eff_only )
         const auto player_local_temp = g->get_temperature( g->u.pos() );
         /* windpower defined in internal velocity units (=.01 mph) */
         double windpower = 100.0f * get_local_windpower( weatherPoint.windpower + vehwindspeed,
-                           cur_om_ter, g->is_sheltered( g->u.pos() ) );
+                           cur_om_ter, g->u.pos(), g->is_sheltered( g->u.pos() ) );
```

Comment:
In `player` member function, does not need `g->u` at all. can replace line with
`cur_om_ter, pos(), g->is_sheltered( pos() ) );` which will also allow it to work nicely with NPC which inherit from `player`